lib/engine/game/g_1849/stock_market.rb

```diff
@@ -38,7 +38,14 @@ def up(corporation, coordinates)
 
         def right(corporation, coordinates)
           price = share_price(coordinates).price
-          return super if !BLOCKED_RIGHT_PRICES.include?(price) || @game.phase.status.include?('blue_zone')
+
+          # Lower two blocked right prices push stock marker UP if not in blue zone
+          if !@game.phase.status.include?('blue_zone') && BLOCKED_RIGHT_PRICES.slice(0, 2).include?(price)
```

Comment:
Let's not slice the constant. The behavior should work for all the block_right_prices. Up will bounce off the ceiling for the highest stock price.

Reply:
Assuming you mean there's already logic further up the chain that will force it to bounce off the ceiling.